lib/components/atoms/Select/Select.js

```diff
@@ -50,6 +50,7 @@ Select.defaultProps = {
   placeholder: '',
   className: '',
   elementLocator: '',
+  name: 'optionName',
```

Comment:
Name should not be part of default props also don't like how we are setting empty default values, please fix it too

Reply:
Updated.